frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -0,0 +1,307 @@
+<template lang='pug'>
+table.table.table-in-card.c-payments(:class='{"c-payments-edit": paymentsType === "edit"}')
+  thead
+    tr
+      th(v-if='paymentsType === "edit"')
+        input.input.c-checkbox(
+          @change='checkAllpayment'
+          type='checkbox'
+        )
+      th {{ titles.one }}
+      th.c-payments-amount {{ titles.two }}
+      th.c-payments-date {{ titles.three }}
+
+  tbody
+    tr(
+      v-for='(payment, index) in payments'
+      :key='index'
+    )
+      td(v-if='paymentsType === "edit"')
+        input.input.c-checkbox(type='checkbox' v-model='payment.checked')
+      td
+        .c-user
+          avatar-user.c-avatar(:username='payment.to' size='xs')
```

Comment:
`:username='payment.to'` isn't working... it should be sent the username, not the display name.

Reply:
It sending the username I believe..

Reply:
I just tested and it's working. You were probably looking at the fake data in the popup